src/main/java/org/fenixedu/academic/ui/spring/AcademicBaseController.java

```diff
@@ -0,0 +1,69 @@
+package org.fenixedu.academic.ui.spring;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+public class AcademicBaseController {
```

Comment:
This pull request was made following the meeting on 24/04/2015. 

While there is no common service in FenixEdu for handling the messages, the common features for adding info/warning/error messages is defined in a common code base so that it can be reused in different controllers.

Creating an abstraction for handling the messages can be used in order to pass the messages also when a redirect is made using flashattributes transparently.